content/docs/index.md

```diff
@@ -2,24 +2,31 @@
 title: Overview
 ---
 
-Kestra is an orchestration and scheduling platform that helps you build, run, schedule, and monitor complex pipelines. It also makes it easy to [create a flow](./05.developer-guide/01.flow.md), requiring only a few lines of YAML. It handles all other complexities like deploying, scaling, monitoring, and error handling.
+Kestra is an open-source, event-driven orchestrator that simplifies data operations and improves collaboration between engineers and business users. By bringing Infrastructure as Code best practices to data pipelines, Kestra allows you to build reliable workflows and manage them with confidence.
 
-![Kestra UI](/ui.gif)
+Kestra allows you to [create flows](./05.developer-guide/01.flow.md) directly from the UI in just a few lines of code. Thanks to the declarative YAML interface for defining orchestration logic, everyone who benefits from analytics can participate in the data pipeline creation process. 
+
+![video-intro](https://kestra.io/video.gif)
+
+
+The UI automatically adjusts the YAML definition any time you make changes to a workflow from the UI or via an API call. Therefore, the orchestration logic is defined declaratively in code, even if some workflow components are modified in other ways.
+
```

Comment:
maybe then "stored" instead of "defined"?

"The UI automatically adjusts the YAML definition any time you make changes to a workflow from the UI or via an API call. Therefore, the orchestration logic is stored declaratively in code, even if some workflow components are modified in other ways."

Reply:
OK then